projects/tad/index.md

```diff
@@ -0,0 +1,84 @@
+|      | Security               | Data encryption is used for sensitive information at rest and in transit. | C        |
+|      | Security               | Regular security audits and penetration testing are conducted. | S        |
+|      | Security               | The project allows for regular security audits and penetration testing to identify vulnerabilities and ensure system integrity. | C        |
+|      | Security               | The tool implements backup functionality to ensure data availability in case of incidents. | C        |
+|      | Compatibility          | The tool is compatible with existing systems and infrastructure. | M        |
+|      | Compatibility          | The tool supports industry-standard data formats and protocols. | M        |
+|      | Compatibility          | The tool operates seamlessly on supported operating systems and hardware platforms. | M        |
+|      | Compatibility          | The tool integrates with existing security solutions.        | C        |
+|      | Compatibility          | The tool supports commonly used data formats (e.g., CSV, Excel, JSON) for easy data exchange with other systems and tools. | S        |
+|      | Data Management        | The tool supports saving progress.                           | S        |
+|      | Workflow Management    | It is possible to define and automate workflows for repetitive tasks. | C        |
+|      | Integration            | The tool offers pre-built connectors or low-code/no-code integration options to simplify the integration process. | C        |
+|      | Accessibility          | The tool is accessible to users with disabilities, following relevant accessibility standards (e.g., WCAG). | S        |
+|      | Portability            | The tool support a range of operating systems (e.g., Windows, macOS, Linux) commonly used within an organization. | S        |
+|      | Portability            | The tool minimizes dependencies on specific hardware or software configurations, promoting flexibility. | S        |
+|      | Portability            | The tool offers a cloud-based deployment option or be compatible with cloud environments for scalability and accessibility. | S        |
+|      | Portability            | The tool adheres to relevant cloud security standards and best practices. | S        |
+|      | Deployment             | The tool has an easy and user-friendly installation and configuration process. | S        |
+|      | Deployment             | The tool has on-premise or cloud-based deployment options to cater to different organizational needs and infrastructure. | S        |
+|      | Legal & Compliance     | The tool is compliant with relevant legal and regulatory requirements. | S        |
+|      | Legal & Compliance     | The tool adheres to (local) data privacy regulations like GDPR and CCPA, ensuring the protection of user data. | S        |
```

Comment:
If so, I'd argue the other way around (keep this one, drop the other one)

Reply:
This differse from just sharing data. It also tackles how to deal with storing data for example.. user sensitive data in an 'open' database (f.e. easy to guess login) or otherwise 'exposed'.. so I'd like to keep it.